plugins/uefi/fu-plugin-uefi.c

```diff
@@ -566,22 +571,25 @@ fu_plugin_uefi_test_secure_boot (FuPlugin *plugin)
 	fu_plugin_add_report_metadata (plugin, "SecureBoot", result_str);
 }
 
-static gboolean
-fu_plugin_uefi_delete_old_capsules (FuPlugin *plugin, GError **error)
+gboolean
+fu_plugin_update_prepare (FuPlugin *plugin,
+			  FwupdInstallFlags flags,
+			  FuDevice *device,
```

Comment:
Don't we need to check if device plugin in uefi? This will get run for a dfu update for instance... Maybe it makes more sense in FuUefiDevice->updateprepare?

Reply:
I don't see a problem moving it into the FuUefiDevice instead.  It will actually allow `fwupdate` to be able to clear old files too then.